src/main/cpp/t03_points.cpp

```diff
@@ -23,10 +23,35 @@
 
 #include "t03_points.h"
 #include <iostream>
-
+#include <vector>
+#include <algorithm>
 
 using namespace std;
 
+struct point {
+ int x, y, distance;
+};
+
+bool cmp ( point a, point b) {
+  return a.distance < b.distance;  
+}
+
 int t03_points() {
-    
+ int n, i, temp1, temp2, dis;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'dis' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4513205)